Code/Editor/Core/EditorActionsHandler.cpp

```diff
@@ -146,6 +183,9 @@ void EditorActionsHandler::Initialize(MainWindow* mainWindow)
     AzToolsFramework::EditorEntityContextNotificationBus::Handler::BusConnect();
     AzToolsFramework::ToolsApplicationNotificationBus::Handler::BusConnect();
     AzToolsFramework::ViewportInteraction::ViewportSettingsNotificationBus::Handler::BusConnect(DefaultViewportId);
+
+    m_editorViewportDisplayInfoHandler = new EditorViewportDisplayInfoHandler();
```

Comment:
question: This is likely moot now given @burelc-amzn's recent change that makes `aznew` obsolete but I wanted to check if we should use that here? (maybe we could do an audit to delete references to it if it's no longer required in a separate PR - this suggestion is for Chris 😛)

Reply:
That's what I did at first but VS gave me errors, so I just resorted to new.

Reply:
Hmm maybe was missing an include, shouldn't matter either way soon 👍